Header header
uint8 cmdtype
uint8 control_cmd
float32 x
float32 y
float32 z
# network data health
uint8 health_flag
